Package.swift

```diff
@@ -305,17 +325,17 @@ func checkSDL2Installed() -> Bool {
 func getGtk4MinorVersion() -> Int? {
     #if os(Windows)
         guard let pkgConfigPath = ProcessInfo.processInfo.environment["PKG_CONFIG_PATH"],
-            case let tripletRoot = URL(fileURLWithPath: pkgConfigPath, isDirectory: true)
-                .deletingLastPathComponent().deletingLastPathComponent(),
-            case let vcpkgInfoDirectory = tripletRoot.deletingLastPathComponent()
-                .appendingPathComponent("vcpkg").appendingPathComponent("info"),
-            let installedList = try? FileManager.default.contentsOfDirectory(
-                at: vcpkgInfoDirectory, includingPropertiesForKeys: nil
-            )
-            .map({ $0.deletingPathExtension().lastPathComponent }),
-            let packageName = installedList.first(where: {
-                $0.hasPrefix("gtk_") && $0.hasSuffix("_\(tripletRoot.lastPathComponent)")
-            })
+              case let tripletRoot = URL(fileURLWithPath: pkgConfigPath, isDirectory: true)
+              .deletingLastPathComponent().deletingLastPathComponent(),
+              case let vcpkgInfoDirectory = tripletRoot.deletingLastPathComponent()
+              .appendingPathComponent("vcpkg").appendingPathComponent("info"),
+              let installedList = try? FileManager.default.contentsOfDirectory(
+                  at: vcpkgInfoDirectory, includingPropertiesForKeys: nil
+              )
+              .map({ $0.deletingPathExtension().lastPathComponent }),
+              let packageName = installedList.first(where: {
+                  $0.hasPrefix("gtk_") && $0.hasSuffix("_\(tripletRoot.lastPathComponent)")
+              })
```

Comment:
Indent these as per before the changes (running `format_and_lint.sh` should hopefully fix it)

Reply:
I tried running the content of the command on my Windows machine but it didn't seems to follow the rules properly, I simply reverted the code. Sorry about the formatting I didn't catch it when I did the commit, I use SwiftFormat on VSCode